src/nidmm/system_tests/test_system_nidmm.py

```diff
@@ -146,7 +146,8 @@ def test_multi_point_acquisition(device_info):
         session.configure_multi_point(4, 2, nidmm.SampleTrigger.IMMEDIATE, 0)
         session.configure_measurement_digits(nidmm.Function.DC_VOLTS, 1, 5.5)
         measurements, numberOfMeasurements = session.read_multi_point(-1, 8)
-        print(measurements)
+        for measurement in measurements:
```

Comment:
I agree this isn't related to this PR. 

How should we deal with these type of small improvements though? It seems like a lot of overhead to try create a new branch, make a change like this, create a PR and then wait for it to be approved.